core/receiver/filter.go

```diff
@@ -2,4 +2,6 @@ package receiver
 
 type Filter struct {
 	ReceiverIDs []uint64
+	Labels      map[string]string
```

Comment:
> I couldn't find the code for filtering receivers by labels, can you help point me to that change? @mabdh

- added labels as filter here
- made changes in proton as well https://github.com/goto/proton/pull/39
- you can check in `store/postgres/receiver.go` we are using jsonb path to filter